src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,31 @@
+package pages;
+
+import io.qameta.allure.Step;
+import org.openqa.selenium.*;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.interactions.Actions;
+
+public class HomePage extends AbstractPage {
+
+    private final By signInBtnLocator = By.linkText("Sign In");
+    private final By topsLinkLocator = By.linkText("Tops");
+    private final By womenLinkLocator = By.xpath("//nav[@class='navigation']//span[contains(text(), 'Women')]");
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+
+    @Step("Going to Registration Page")
+    public void openSignInPage() {
+        driver.findElement(signInBtnLocator).click();
+    }
+
+    @Step("Going to Women Tops Page")
+    public void clickWomenTops() {
+        Actions actions = new Actions(driver);
+        actions.moveToElement(driver.findElement(womenLinkLocator)).perform();
```

Comment:
why are you mouse actions, not the normal click? it's the correct approach if the normal element click is not working.
In case it's the case you need try with locator to your target element and child elements.
Waiting for  clickability of element could also help.

Reply:
That is the logic of the test: you need to hover over the mouse so that the dropdown appears and then you click "Tops". If you click on "Women" you are redirected to another page

Reply:
Got it. In this case all is correct